Vagrantfile

```diff
@@ -127,6 +127,11 @@ Vagrant.configure("2") do |config|
     main.vm.provision "Run local Docker registry script for all nodes", type: "shell",
                       path: "image/scripts/dev/docker-registry/all_node_registry.sh",
                       args: [fqdn, main_k3s_ip_address]
+
+    main.trigger.before [:halt, :reload] do |trigger|
+      trigger.info = "Running k3s-killall.sh to shut down k3s..."
```

Comment:
I suggest to skip the filename if it changes in the future. But that is just an impl. detail.